internal/pkg/config/config.go

```diff
@@ -7,42 +7,47 @@ import (
 	"buf.build/go/protoyaml"
 	attestation_policy_proto "github.com/cofide/cofide-api-sdk/gen/go/proto/attestation_policy/v1alpha1"
 	config_proto "github.com/cofide/cofide-api-sdk/gen/go/proto/config/v1alpha1"
+	pluginspb "github.com/cofide/cofide-api-sdk/gen/go/proto/plugins/v1alpha1"
```

Comment:
Nit: this alias could be `plugins_proto`, similar to how others are named perhaps?

Reply:
The underscores in the proto import renames [aren't idiomatic](https://go.dev/blog/package-names#package-paths).

> When renaming an imported package, the local name should follow the same guidelines as package names (lower case, no under_scores or mixedCaps).

I figured for existing packages we can keep them for now but any new ones should probably use a different scheme. I was thinking of doing a big find/replace for the older ones when things are a bit quieter.

The `pb` suffix seems to be quite commonly used, at least in Google, and is nice and short (https://pkg.go.dev/google.golang.org/protobuf/types/known/anypb, https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb). Spire uses a <package><version> format, e.g. `bundlev1`.